docs/openapi.yaml

```diff
@@ -10,7 +10,7 @@ servers:
     description: Local
 info:
   title: Stacks 2.0 Blockchain API
-  version: '1.0.0'
+  version: 'STACKS_API_VERSION'
```

Comment:
I suspect this change is what's breaking the vercel CI docs from building -- do you know if this stub is filled by the time the root package `build:docs` command is ran?

Reply:
Good point, it's not, I'll fix that 👍